Update repetition time proposal to fit with new formatting for bids specification

KirstieJane opened the pull request:
I've had a re-format to try to line up this proposal as text that could be submitted to https://github.com/bids-standard/bids-specification as a PR (when we're ready!)

@agahkarakuzu - I've done a big re-arrange of your words. Some of it is to try to make certain definitions a bit more terse (eg in the tables) or to re-format slightly differently. I *hope* I haven't changed the meaning.

Would you please review and let me know if it makes sense??

Gilles86 committed:
Added some stuff about branches to the README

KirstieJane committed:
Merge branch 'repetitiontime' of https://github.com/agahkarakuzu/bep001 into repetitiontime

KirstieJane committed:
halfway through re-formatting

KirstieJane committed:
Tiny tidy up

KirstieJane committed:
Format change to fMRI section

KirstieJane committed:
Merge branch 'repetitiontime' of https://github.com/agahkarakuzu/bep001 into repetitiontime

agahkarakuzu commented on Proposal_RepetitionTime.md:
```diff
@@ -1,94 +1,110 @@
 
-What follows is the text from the current BIDS specification along with the suggested additions.
-Additions and replacements are rendered twice: in easy to read markdown and in "code diff" format to make it easy to see the proposed changes.
+*What follows is the text from the current BIDS specification with the suggested additions inclued.*
```
Typo inclued --> included

agahkarakuzu commented on Proposal_RepetitionTime.md:
```diff
@@ -1,94 +1,110 @@
+Common examples of prepulses are an inversion prepulse and a magnetization transfer prepulse.
+An inversion RF pulse applied prior to the excitation pulse may be used to prepare a desired tissue contrast, typically to create higher levels of T1 weighting.
+A magnetization transfer prepulse is an off-resonant RF pulse that is applied prior to the excitation pulse to saturate protons associated with macromolecules.
+(Note that there can be multiple off-resonanct RF pulses.)
```
Typo off-resonanct->off-resonant

agahkarakuzu commented on Proposal_RepetitionTime.md:
```diff
@@ -1,94 +1,110 @@
+
+An example of an anatomical acquisition sequence that would require the `RepetitionTimeExcitation` metadata field is .......
+
+Some anatomical acquisitions need to record two measurements of time.
```
Maybe we can reword this for clarity.

KirstieJane committed:
Fix typos

KirstieJane commented:
Addressed your feedback @agahkarakuzu! Thank you!

There's an example missing that I need some help with - so if you have any time to suggest some text for that it would be amazing 👾
